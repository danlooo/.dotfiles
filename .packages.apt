# List of essential apt packages

docker
vim-gtk3
npm
r-base
python3
python3-pip
fonts-powerline
python3-proselint
