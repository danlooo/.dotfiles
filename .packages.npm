#  List of essential npm packages

alex
write-good
