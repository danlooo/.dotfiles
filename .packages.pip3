#  List of essential pip packages

powerline-shell
black
pylint
