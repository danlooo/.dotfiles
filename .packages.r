#  List of essential R packages

tidyverse
lintr
styler
