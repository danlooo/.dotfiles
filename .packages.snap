#  List of essential snap packages

languagetool
